src/ukify/ukify.py

```diff
@@ -1695,7 +1695,7 @@ def inspect_section(
 
     if ttype == 'text':
         try:
-            struct['text'] = data.decode()
+            struct['text'] = data.decode(errors='replace').replace('\x00', '')
```

Comment:
What's the idea behind adding `errors='replace'` here? Could this also be achieved by changing the order of operations, e.g. 
```suggestion
            struct['text'] = data.replace(b'\0', '').decode()
```

Also, since this is padding, shouldn't it be only at the end, i.e. `rstrip` or `strip` should do?

Reply:
> What's the idea behind adding `errors='replace'` here? Could this also be achieved by changing the order of operations, e.g.

Ok

> Also, since this is padding, shouldn't it be only at the end, i.e. `rstrip` or `strip` should do?

It's at end of a line but not at end of the string from my experiment.